source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation.py

```diff
@@ -116,6 +116,10 @@ def is_fixed_base(self) -> bool:
         """Whether the articulation is a fixed-base or floating-base system."""
         return self.root_physx_view.shared_metatype.fixed_base
 
+    @property
+    def num_instances(self) -> int:
```

Comment:
Does this need a doc string?

My interpretation of "num_instances" in this case is the number of replications of the articulation. Is this the correct interpretation? Can this be different than the number of environments?

Reply:
For most learning setup, "num_instances" = "num_envs". At least that's the main underlying idea. However, for a single environment case, the view can be used to capture two instances of the same robot in that environment.

Scaling it to N environments with M same robots in each remains a discussion point because all the tensor shapes change in that case. We can think of adding it if we see a huge perf gain by doing so, i.e. one view capturing M robots vs. one view per robot.

Reply:
That sounds reasonable. What about learning setups for multi-robot collaboration (i.e. two arm manipulation) do we set it up such that the two robots are viewed as one?

Reply:
There are two separate robot objects.